sql/v1.06_db_changes/21711_21718_missingdrops.sql

```diff
@@ -0,0 +1,10 @@
+-- For Quest: Rowan's Request 1 - 21711
+INSERT INTO drop_data (`dropperid`, `itemid`, `minimum_quantity`, `maximum_quantity`, `questid`, `chance`)
+VALUES 
+(1110130, 4032317, 1, 1, 21711, 30000); -- Dejected Green Mushroom | Dejected Green Mushroom Puppet | Rowan's Request 1
```

Comment:
The quest number is actually 21717. Didn't check the quest number properly, sorry.

Reply:
I've updated the SQL